pages/common/mupdf.md

```diff
@@ -0,0 +1,32 @@
+# mupdf
+
+> MuPDF is a lightweight PDF viewer (also supports XPS and E-Book formats).
+> More information: <https://www.mupdf.com>.
+
+- Open a PDF on the first page:
+
+`mupdf {{filename}}`
+
+- Open a PDF on page 3:
+
+`mupdf {{filename}} {{3}}`
+
+- Open a password secured PDF:
+
+`mupdf -p {{password}} {{filename}}`
+
+- Open a PDF with an initial zoom level, specified as DPI, of 72:
+
+`mupdf -r {{72}} {{filename}}`
+
+- Open a PDF with inverted color:
+
+`mupdf -I {{filename}}`
+
+- Open a PDF tinted red FF0000 (hexadecimal color syntax RRGGBB):
```

Comment:
How about the standard web color format...
```suggestion
- Open a PDF tinted red #FF0000 (hexadecimal color syntax RRGGBB):
```

Reply:
As a non-web developer I don't come across this syntax often, in other environments we would write 0xFF0000 instead. But I feel your suggestion is valid, I will accept the commit suggestions :)

Reply:
I agree. I mainly suggested this because plain FF0000 doesn't look quite good to me, and 0xFF0000 doesn't particularly strike me as a color :)

Beside, when dealing with creating/editing PDFs, one will probably encounter hexadecimal colors quite frequently.